pages/common/sgpt.md

```diff
@@ -0,0 +1,28 @@
+# sgpt
+
+> A command-line productivity tool powered by OpenAI's GPT models.
+> More information: <https://github.com/TheR1D/shell_gpt#readme>.
+
+- Use it as normal `Search Engine`, asking for mass of the sun:
+
+`sgpt "mass of the sun"`
+
+- Execute `Shell` commands, apply chmod 444 to all files in current directory:
+
+`sgpt --shell "make all files in current directory read only"`
+
+- Generate `Code`, solving classic fizz buzz problem:
+
+`sgpt --code "Solve fizz buzz problem using Python"`
+
+- Create a `Chat session` by giving a unique session name ssn:
+
+`sgpt --chat ssn "please remember my favorite number: 4"`
+
+- Start a `REPL` (Read–eval–print loop) session:
+
+`sgpt --repl TEXT`
+
+- Print Shell-GPT help manual:
```

Comment:
```suggestion
- Display help:
```
We use `Display help` as a generic placeholder for `--help` command in most of our pages.